example_graceful_shutdown_test.go

```diff
@@ -169,5 +169,5 @@ func Example_gracefulShutdown() {
 	// Received SIGINT/SIGTERM; initiating soft stop (try to wait for jobs to finish)
 	// Received SIGINT/SIGTERM again; initiating hard stop (cancel everything)
 	// Job cancelled
-	// JobExecutor: Job errored; retrying
+	// jobexecutor.JobExecutor: Job errored; retrying
```

Comment:
I modified the service naming logic a little below so that we produce names like `riverlog.Middleware`. Top-level River constructs don't include the package so it just looks like `Client` (with the idea that it's in River being implied).